source/_includes/code/navigation/vertical-navigation/code.md

```diff
@@ -1,4 +1,26 @@
-<p>Jump to <a href="#example-code-1">With Primary Menu Icons</a> or <a href="#example-code-2">Without Primary Menu Icons</a></p>
+<p>Jump to <a href="#example-code-3">Masthead Design</a>, <a href="#example-code-1">With Primary Menu Icons</a> or <a href="#example-code-2">Without Primary Menu Icons</a></p>
+<h2 id="example-code-3">Masthead Design</h2>
+<div class="example-pf">
+  <iframe src="{{ site.baseurl}}/pattern-library/navigation/vertical-navigation/vertical-navigation-masthead.html"
+          width="100%" height="650px;" scrolling="no" seamless></iframe>
+</div>
+<p><a href="{{ site.baseurl}}/pattern-library/navigation/vertical-navigation/vertical-navigation-masthead.html" target="_blank">View full page example</a></p>
+<div class="row">
+  <div class="col-md-12">
+    <p class="reference-markup"><a class="collapse-toggle" data-toggle="collapse" aria-expanded="true" aria-controls="markup-3" href="#markup-3">Reference Markup</a></p>
+    <div class="collapse in" id="markup-3">
+      <pre class="prettyprint">{% capture markup_include %}
+<html class="layout-pf layout-pf-fixed">
+...
```

Comment:
I would suggest removing the `...`

Reply:
@dabeng I also noticed there's a wide gap between where the `scripts` end and where the `nav` code begins. Is there any way to fix that gap or is that how the code is included in the reference markup?